### DISCUSSION TITLE
lazer crashes when i plug in a wired controller. not sure whats happening here

### DISCUSSION THREAD

**Discussion by xxMito** [ppy/osu#17357] [Q&A]
my osu lazer (version 2022.319.0-lazer) is crashing when i plug in my wired ps4 controller.

the game doesn't boot up again while it's plugged in. 
the game keeps running in the background.
when i unplug the controller, the game starts up again. 

sorry if this is not a bug, i just wanted to report it somewhere. 
im not even sure if controllers are supported. i wanted to try diva ruleset with it

[performance.log](https://github.com/ppy/osu/files/8310334/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8310335/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8310336/updater.log)
[database.log](https://github.com/ppy/osu/files/8310337/database.log)
[network.log](https://github.com/ppy/osu/files/8310338/network.log)

**Comment by peppy** [comment]
There's no crash in the log files. Can you also check event viewer for any errors?

**Reply by xxMito** [reply]
it showed application hang and .net runtime errors
quick update: i just found that the game doesn't really crash, it hangs for a really long time. 
![image](https://user-images.githubusercontent.com/43375002/159159011-dc812e9c-e368-46e0-b72b-e078c17f59cc.png)
![image](https://user-images.githubusercontent.com/43375002/159159018-0e7f6193-7b9b-4a78-ba4e-66c0d45f79a2.png)
![image](https://user-images.githubusercontent.com/43375002/159159542-8a7ed0e6-9147-4d85-a815-826b6379c24f.png)

**Reply by peppy** [reply]
Judging by the nature of these error reports, this is most likely going to be an issue at SDL's end. [This issue](https://github.com/libsdl-org/SDL/issues/5150) may be related.
